db.go

```diff
@@ -32,8 +34,31 @@ func WithDiscardUnknownColumns() DBOption {
 	}
 }
 
+type SQLRepo interface {
```

Comment:
What is SQLRepo? It sounds strange... Perhaps `SQLDB`?

Reply:
https://threedots.tech/post/repository-pattern-in-go/

Ideally this is to show that it is a  Repo pattern followed.

Initially I had it as SQLDB but found harder to read since all letters are caps

Reply:
Plus the whole point was to give an independency for developers  from database/sql, so that dbresolvers, dbloggers based on database/sql can be integrated to bun